common/utils.go

```diff
@@ -0,0 +1,12 @@
+package common
+
+import (
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/shutter-network/rolling-shutter/rolling-shutter/medley/identitypreimage"
+	"github.com/shutter-network/shutter/shlib/shcrypto"
+)
+
+func ComputeIdentity(prefix []byte, sender common.Address) *shcrypto.EpochID {
```

Comment:
Same comment on the identity calculation here, it should be a keccak hash of the prefix + sender, similar to what we have in the contract.